internal/pkg/build/sources/conveyorPacker_zypper.go

```diff
@@ -74,6 +79,124 @@ func (cp *ZypperConveyorPacker) Get(b *types.Bundle) (err error) {
 	// add aaa_base to start of include list by default
 	include = `aaa_base ` + include
 
+	// get mirrorURL, OSVerison, and Includes components to definition
+	osversion, osversion_ok := cp.b.Recipe.Header["osversion"]
+	mirrorurl, mirrorurl_ok := cp.b.Recipe.Header["mirrorurl"]
+	updateurl, updateurl_ok := cp.b.Recipe.Header["updateurl"]
+	sleproduct, sleproduct_ok := cp.b.Recipe.Header["product"]
+	sleuser, sleuser_ok := cp.b.Recipe.Header["user"]
+	sleregcode, sleregcode_ok := cp.b.Recipe.Header["regcode"]
+	slepgp, slepgp_ok := cp.b.Recipe.Header["productpgp"]
+	sleurl, sleurl_ok := cp.b.Recipe.Header["registerurl"]
+	slemodules, slemodules_ok := cp.b.Recipe.Header["modules"]
+	cnt := -1
+	if tmp, ok := cp.b.Recipe.Header["otherurl0"]; ok {
+		otherurl[0] = tmp
+		cnt = 1
+	} else {
+		if tmp, ok := cp.b.Recipe.Header["otherurl1"]; ok {
+			otherurl[0] = tmp
+			cnt = 2
+		}
+	}
+	for i := 1; cnt > 0 && i < 20; i++ {
+		numS := strconv.Itoa(cnt)
+		if tmp, ok := cp.b.Recipe.Header["otherurl"+numS]; ok {
+			otherurl[i] = tmp
+			cnt++
+		} else {
+			cnt = -1
+		}
+	}
+	regex := regexp.MustCompile(`(?i)%{OSVERSION}`)
+
+	if sleproduct_ok || sleuser_ok || sleregcode_ok {
+		if !sleproduct_ok || !sleuser_ok || !sleregcode_ok {
+			return fmt.Errorf("For installation of SLE 'Product', 'User' and 'Regcode' need to be set\n")
+		}
+		if !osversion_ok {
+			return fmt.Errorf("Invalid zypper header, OSVersion always required for SLE")
+		}
+		if !slepgp_ok && !mirrorurl_ok {
+			return fmt.Errorf("No 'SLEpgp' and no 'InstallURL' defined in bootstrap definition\n")
+		}
+		suseconnectPath, err = exec.LookPath("SUSEConnect")
+		if err != nil {
+			return fmt.Errorf("SUSEConnect is not in PATH: %v", err)
+		}
+
+		array := strings.SplitN(osversion, ".", -1)
+		osmajor := array[0]
+		iosmajor, err = strconv.Atoi(osmajor)
+		if err != nil {
+			return fmt.Errorf("OSVersion has wrong format %v", err)
+		}
+		osminor := ""
+		if len(array) > 1 {
+			osminor = "." + array[1]
+		}
+		if iosmajor > 12 && !mirrorurl_ok {
+			return fmt.Errorf("For SLE version > 12 'MirrorURL' must be defined and point to the installer\n")
+		}
+		osservicepack := ""
+		tmp, err := strconv.Atoi(array[1])
+		if err != nil {
+			return fmt.Errorf("Cannot convert minor version string to integer: %v", err)
```

Comment:
To be consistent (and its good practice), you should't use caps in return errors, this should be:

```suggestion
			return fmt.Errorf("cannot convert minor version string to integer: %v", err)
```